dfastmi/batch.py

```diff
@@ -2032,6 +2026,27 @@ def check_configuration_v1(rivers: RiversObject, config: configparser.ConfigPars
                 
     return True
 
+def get_reach(rivers, config):
+    """
+
```

Comment:
will this be public or private or will reach be moved to a seperate class?
Based on that documentation should be added.

Reply:
I agree, but batch.py is not in scope of this issue. I only refactor code to get it working again. Will pickup the good comments later in issue DFAST-103 or DFAST-104

Reply:
Discussed, we will leave these comments for a later refactoring.
When a new issue is created for this, the number will be disclosed here.